### ISSUE TITLE
Doesn't work with the new Obsidian editor

### ISSUE THREAD

**Issue by gnatyuk616** [konodyuk/obsidian-text-expander#15]
I'm using Obsidian v0.13.8.
Text expander stopped working, the Tab command just indent the {{trigger}} without actual substitution.
When I switched to the Legacy Editor at the Editor section, everything starts working again.

**Comment by uwe999** [comment]
+1

**Comment by ThisIsDoubleC** [comment]
+1 my text just gets tabbed and becomes red

**Comment by konodyuk** [comment]
The plugin relies on CM5 (considered legacy since v13.0.0) and its failure is expected when used with CM6. Unfortunately, I have no plans to upgrade the plugin to CM6 in its current condition. I thought about dropping shell support in favor of JS-defined scripts to support mobile devices. Since this change requires full refactoring, it could be the right point to move to CM6, but I cannot give any timeline right now.

**Comment by Shirobachi** [comment]
I agree, sorry to say, but I am not into legacy mode, and also I would love to be able, use this plugin. Are you considering updating the plugin, so it can run in new obsidian without legacy mode?

**Comment by andyholst** [comment]
Use ChatGPT to port aka. refactor the code @konodyuk to save time.

**Comment by grhawk** [comment]
IMHO, I think refactoring the plugin to use JS would not be such a good approach. If you know how to use JS you could probably achive similar results with different way (for example having JS scripts inside obsidian). I think one of the very good point of this plugin is the possibility to use directly BASH as a backend.
